modules/file-system/netapp-storage-pool/versions.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 6.45.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.0"
+    }
+  }
+
+  provider_meta "google" {
+    module_name = "blueprints/terraform/hpc-toolkit:netapp-storage-pool/v1.70.0"
+  }
+  provider_meta "google-beta" {
+    module_name = "blueprints/terraform/hpc-toolkit:netapp-storage-pool/v1.70.0"
+  }
+
+  # Require Terraform version 1.9.0 or higher, since that added support
+  # for variable conditions. See https://support.hashicorp.com/hc/en-us/articles/43291233547027-Error-Invalid-reference-in-variable-validation-in-Terraform-versions-prior-to-1-9
+  required_version = ">= 1.9.0"
```

Comment:
I would not go above the currently supported version which is 1.5.7. This is *very* important. Please make the appropriate fixes to work on 1.5.7